mirror/create_mirror_rpm.sh

```diff
@@ -33,7 +33,10 @@ mkdir -p $RPM_REPO_DIR
 
 cd $RPM_REPO_DIR
 cp /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7 $RPM_REPO_DIR
-cp /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release $RPM_REPO_DIR
+if [ "x$DISTRO" == "xrhel" ]; then
```

Comment:
Does this not cause problems when the bootstrap scripts try to import that key?
https://github.com/pndaproject/pnda-aws-templates/blob/develop/bootstrap-scripts/package-install.sh#L33